git_machete/annotation.py

```diff
@@ -0,0 +1,59 @@
+import re
+from typing import Callable, Optional
+
+from git_machete import utils
+
+
+class Qualifiers:
+    rebase: Optional[bool]
+    merge: Optional[bool]
+    push: Optional[bool]
+
+    def __init__(self, annotation: str):
+        self._annotation = annotation
+        self._annotation_without_qualifiers = annotation
+        self._rebase_text = ''
+        self._push_text = ''
+        self.rebase = None
+        self.push = None
```

Comment:
Hmmm maybe just assume `True` for both? so as not to risk confusing `None` and `False` in constructs like `if push:` 🤔 and change the types to non-optional `bool`

Reply:
@PawelLipski I did not want to assume `True` since it can possibly cause bugs in the future

Reply:
But i see that it is assumed already in the `traverse()` function so i guess it might be better to have it in qualifiers